packages/fxa-auth-server/lib/senders/emails/renderer.ts

```diff
@@ -0,0 +1,21 @@
+import ejs = require('ejs');
+import mjml2html = require('mjml');
+import * as templates from './templates';
+
+export const context = {
+  buttonText: 'Sync device',
+  onDesktopOrTabletDevice: true,
+  anotherDeviceURL:
+    'http://localhost:3030/connect_another_device?utm_medium=email&utm_campaign=fx-cad-reminder-first&utm_content=fx-connect-device',
+  iosURL: '',
+  androidURL: '',
+};
```

Comment:
Looks like this context isn't being used any more and can be removed (context is coming from the function call now)

Reply:
I am actually using this context object in both the scripts [`preview-in-maidev`](https://github.com/Yogita98/fxa/blob/EJS%2BMJML-solution/packages/fxa-auth-server/lib/senders/emails/preview-in-maildev.ts#L3) and [`render-to-disk`](https://github.com/Yogita98/fxa/blob/EJS%2BMJML-solution/packages/fxa-auth-server/lib/senders/emails/render-to-disk.ts#L4) and couldn't really find a better place to import it from but I can definitely either create a separate file or declare the object individually in both the files. Let me know your thoughts

Reply:
Oh my bad I didn't see that it was exported. Never mind!